MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile

```diff
@@ -3,6 +3,7 @@ use_frameworks!
 
 target 'PodInstall' do
   # The branch of MapboxNavigation and MapboxNavigation will be replaced by Bitrise to use the current branch.
+  pod 'MapboxDirections.swift', :git => 'https://github.com/mapbox/MapboxDirections.swift.git', :branch => 'jerrad/delenda-duet-dependencies'
```

Comment:
This change and the corresponding change to Podfile.lock need to be updated once mapbox/MapboxDirections.swift#393 lands in master and reverted once MapboxDirections.swift v1.0 is released.